firmware/third_party/CMakeLists.txt

```diff
@@ -42,6 +42,18 @@ file(GLOB_RECURSE SBG_ECOM_SRCS
 )
 list(APPEND SHARED_ARM_BINARY_X86_INCOMPATIBLE_THIRD_PARTY_SRCS ${SBG_ECOM_SRCS})
 
+# Import littlefs
+CPMAddPackage(
+  NAME littlefs
+  GITHUB_REPOSITORY littlefs-project/littlefs
+  GIT_TAG v2.8.0
+  SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/littlefs 
```

Comment:
please consider keeping littlefs in the _deps folder, and using the autogenerated variables to find  the directory (see [here](https://github.com/cpm-cmake/CPM.cmake#usage))
![image](https://github.com/UBCFormulaElectric/Consolidated-Firmware/assets/20777515/00337aac-598a-4445-81db-f81d3c23d462)

Reply:
It seems like I can't put it into `_deps` folder while passing the pull request test. The error message suggests that git was set only to search for a depth of 1.